src/test/ui/lint/issue-54538-unused-parens-lint.rs

```diff
@@ -15,14 +15,14 @@ fn main() {
     }
```

Comment:
Use `#![allow(...)]` and duplicate all `match`es here to exercise `...` in the new ones as well as `..=` in the old ones.